test-result-summary-client/src/Dashboard/Widgets/Graph/ODM.jsx

```diff
@@ -9,14 +9,14 @@ import math from 'mathjs';
 import utils from './utils';
 
 const map = {
-    "Daily-ODM-all": "Daily-ODM Daily-ODM-Linux-PPCLE64 Daily-ODM-openJ9 Daily-ODM-zLinux Daily-ODM-zOS"
+    "Daily-ODM-all": "PerfNext-Sophia-Child PerfNext-Sophia-Master Daily-ODM Daily-ODM-Linux-PPCLE64 Daily-ODM-openJ9 Daily-ODM-zLinux Daily-ODM-zOS"

```

Comment:
Let's not use pipelines with personal names. Ideally, we should be reading the pipeline list from a JSON file that doesn't need to be checked in or we could check in a template. You can use https://github.com/AdoptOpenJDK/openjdk-test-tools/blob/master/TestResultSummaryService/DashboardBuildInfo.json as an example.